ui/js/models/job.js

```diff
@@ -94,6 +94,17 @@ treeherder.factory('ThJobModel', [
         });
     };
 
+    ThJobModel.backfill = function(repoName, pk, config) {
+        config = config || {};
+        var timeout = config.timeout || null;
+
+        return $http.post(ThJobModel.get_uri(repoName)+pk+"/backfill/",
+            {timeout:timeout})
+            .then(function(response) {
```

Comment:
You should delete this .then call because 1) the backfill endpoint doesn't return a job and 2) the code that calls this function doesn't expect it to return a ThJobModel.